src/health-check/main.cpp

```diff
@@ -132,16 +132,14 @@ class HealthCheckerProcess : public ProtobufProcess<HealthCheckerProcess>
   {
     VLOG(1) << "Check passed";
 
-    // Send a healthy status update on the first success,
-    // and on the first success following failure(s).
-    if (initializing || consecutiveFailures > 0) {
-      TaskHealthStatus taskHealthStatus;
```

Comment:
OK, LGTM. If you are not familiar with review board, I could help you to post this patch there.